applications/StructuralMechanicsApplication/custom_elements/base_shell_element.cpp

```diff
@@ -595,4 +595,4 @@ inline std::ostream & operator << (std::ostream& rOStream, const BaseShellElemen
   return rOStream;
 }
 
-} // namespace Kratos.
+} // namespace Kratos.
```

Comment:
please do not make such changes 
This doesn't change the code but it messes up the Git-history, which in turn makes it harder to debug